app/main.py

```diff
@@ -1,31 +1,104 @@
-from fastapi import FastAPI, HTTPException
+from fastapi import FastAPI, HTTPException, Depends
 from starlette.status import HTTP_404_NOT_FOUND
 from models import Oeuvre,Calque,TypeCalque
 from datetime import datetime
 from starlette.staticfiles import StaticFiles
+from starlette.responses import Response
+from starlette.requests import Request
+from sqlalchemy.orm import validates
+from sqlalchemy import Boolean, Column, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base, declared_attr
+from sqlalchemy.orm import Session, sessionmaker
 
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.orm import joinedload
 
-app = FastAPI()
+SQLALCHEMY_DATABASE_URI = "sqlite:///./database_ddale.db"
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URI, connect_args={"check_same_thread": False}
+)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+Base = declarative_base()
+
+# TODO : Ce serait bien de mettre ça dans le fichier models.py.
+# Transcription des classes de models pour les rendre 
+# au même format que la BdD.
+class OeuvreDb(Base):
+    __tablename__ = "oeuvres"
+    id = Column(Integer, primary_key=True, nullable=False)
+    titre = Column(Integer, primary_key=True, nullable=False)
```

Comment:
```suggestion
    titre = Column(Integer, nullable=False)
```
Titre ne devrait pas être une primary key